webrtc.html

```diff
@@ -2054,6 +2054,36 @@
         </dl>
       </section>
     </section>
+
+	<section>
+	<h2>QoS Model</h2>
+
+	<p> Many applications have multiple media flows of the same
```

Comment:
Should this also contain a reference to rtcweb-transport, for the intra-browser prioritization aspect?